terraform/aws/modules/eks-managed-node-groups/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "vpc_security_group_ids" {}
```

Comment:
no blocker, some description for each field will be nice :)

Reply:
Done :)